pixeltable/share/packager.py

```diff
@@ -200,6 +220,92 @@ def __build_tarball(self) -> Path:
                 tf.add(src_file, arcname=f'media/{dest_name}')
         return bundle_path
 
+    def __extract_preview_data(self) -> tuple[dict[str, str], list[list[Any]]]:
+        """
+        Extract a preview of the table data for display in the UI.
+
+        In order to bound the size of the output data, all "unbounded" data types are resized:
+        - Strings are abbreviated as per Formatter.abbreviate()
```

Comment:
Should we send unabbreviated string (and json) data and let the UI decide how much to abbreviate? A single column value that is stored in Postgres is bounded in size.

Reply:
I thought we were allowing large strings in postgres?

What is the limit (in bytes) for strings/JSON?